tests/integration-tests/tests/storage/storage_common.py

```diff
@@ -52,6 +52,39 @@ def get_cluster_subnet_ids_groups(cluster: Cluster, scheduler: str, include_head
     return compute_subnet_ids
 
 
+def test_directory_correctly_shared_between_ln_and_hn(
+    remote_command_executor_head_node, remote_command_executor_login_node, mount_dir
+):
+    logging.info("Testing FS correctly mounted on login nodes")
```

Comment:
Please add a brief comments that describes what the test expect to achieve and which preconditions it expects (if any) ... like
"
It expects an ebs volume to be mounted at <HN_folder> on the HeadNode and exported through NFS
it expects the ebs volume to be mounted at <LN_folder> on the LoginNode through NFS

it will write a file from the HeadNode and a file from the LoginNode in the shared volume

it expects to be able to read both files from the HeadNode and the LoginNode

Reply:
Thanks for reminding. Done!